microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -135,45 +137,47 @@ Define the specific markets in which you plan to offer your extension.  The defa
 
-> [!IMPORTANT]
-> Your developer/organization name, website URL, and support contact details that you submitted during registration are displayed to users on the Microsoft Edge Add-ons website.
+Click the **Save & Continue** button<!-- todo: link, or button? --> to continue to the **Store listings** section.
 
 
 <!-- ========================================================================== -->
 ## Step 5: Add Store listing details for your extension
 
-The information provided in the following section is displayed to users who review your listing on the Microsoft Edge Add-ons website.  Even though some fields are optional, you should provide as much information as possible.  To list your extension in the store, the following details are required.
+The information provided in the following section is displayed to users who review your listing on the Microsoft Edge Add-ons website.  Even though some fields are optional, you should provide as much information as possible.
 
-*   **Description** for each language in your extension package. To support multiple languages, you can use the internationalization API ([chrome.i18n](https://go.microsoft.com/fwlink/?linkid=2167478)).
-*   **Extension Store logo** for each language in your extension package.
+To list your extension in the store, the following details are required:
+* **Display name**.
+* **Description** for each language in your extension package. To support multiple languages, you can use the internationalization API ([chrome.i18n](https://go.microsoft.com/fwlink/?linkid=2167478)).
+* **Extension Store logo** for each language in your extension package.
+* **Short description**.
 
-> [!NOTE]
-> The minimum required store listing details must be filled-out for at least one of the languages mentioned in your extension zip package.  To add or remove languages in your store listing on the Microsoft Edge Add-ons website, use the **Add a language** dropdown list on the **Store listings** webpage.  Additionally, you can duplicate your assets from one language across others using the **Duplicate functionality** button on the **Language details** webpage.
+The minimum required store listing details must be filled-out for at least one of the languages<!-- todo: spoken langs, or coding langs? --> mentioned in your extension zip package.  To add or remove languages in your store listing on the Microsoft Edge Add-ons website, use the **Add a language** dropdown list on the **Store listings** webpage.  Additionally, you can duplicate your assets from one language across others using the **Duplicate functionality** button on the **Language details** webpage.
```

Comment:
spoken langs, or coding langs? - 
"The minimum required store listing details must be filled-out for at least one of the languages mentioned in your extension zip package."

rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/publish-extension?branch=pr-en-us-3330#step-5-add-store-listing-details-for-your-extension